lusSTR/wrappers/snps_convert.py

```diff
@@ -97,7 +95,32 @@ def create_sample_df(indiv_df, output_type, all_col):
     try:
         compiled_table.columns = ["Marker", "Allele 1", "Allele 2", "Height 1", "Height 2"]
     except ValueError:
-        print("Too many alleles!")
+        try:
+            compiled_table.columns = [
+                "Marker",
+                "Allele 1",
+                "Allele 2",
+                "Allele 3",
+                "Height 1",
+                "Height 2",
+                "Height 3",
+            ]
+        except ValueError:
+            compiled_table.columns = [
+                "Marker",
+                "Allele 1",
+                "Allele 2",
+                "Allele 3",
+                "Allele 4",
+                "Height 1",
+                "Height 2",
+                "Height 3",
+                "Height 4",
+            ]
+            if len(compiled_table[compiled_table["Allele 4"].notna()]) > 0:
+                compiled_table = compiled_table.drop(compiled_table.columns[[4, 8]], axis=1)
+        if len(compiled_table[compiled_table["Allele 3"].notna()]) > 0:
+            compiled_table = compiled_table.drop(compiled_table.columns[[3, 6]], axis=1)
```

Comment:
I noticed this problem when running a lot of mix deconvolution in the past few months- the original code cut off any extra alleles included (all likely errors but still important to be included for the EFM software).